maxim/logger/openai/utils.py

```
@@ -122,34 +122,77 @@ def parse_completion(
-                else 0,
-                "completion_tokens": completion.usage.completion_tokens
-                if completion.usage
-                else 0,
-                "total_tokens": completion.usage.total_tokens
-                if completion.usage
-                else 0,
-            },
-        }
+    @staticmethod
+    def parse_completion_from_chunks(
+        chunks: List[ChatCompletionChunk],
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)